### DISCUSSION TITLE
Wget not available is now ERROR, not a WARNING

### DISCUSSION THREAD

**Discussion by tony-sol** [mason-org/mason.nvim#1038] [Q&A] [answered]
According to `mason.nvim/lua/mason/health/init.lua` curl may be replacement for wget, it seems not working

<img width="843" alt="Снимок экрана 2023-02-27 в 15 05 56" src="https://user-images.githubusercontent.com/33671815/221559602-b20c4bc2-9bec-4f9e-a736-ffe748df6656.png">

Did i misconfigure something?

**Comment by williamboman** [comment, marked as answer]
Hey! `terraform-ls` is failing because the installation method changed recently, update `mason.nvim` to fix it. The error message is mentioning `wget` because it's the last utility that is tried.

**Reply by tony-sol** [reply]
I'd updated `mason.nvim` via packer and installed LSPs via mason.
Some had been updated well, but not a `terraform-ls`, with same error about `wget`

**Reply by williamboman** [reply]
Ah I see that now, sorry! There had been quite a few reports about `terraform-ls` specifically so my brain short-circuited. Hm that seems weird. There's an [issue template here](https://github.com/williamboman/mason.nvim/issues/new?assignees=&labels=installation-issue&template=package_installation_form.yaml) for installation issues.

**Reply by tony-sol** [reply]
Problem was found - could not download from hashicorp's origin due to my vpn settings
